packages/backend/src/config/Config.ts

```diff
@@ -79,6 +79,7 @@ export interface ActivityV2Config {
   readonly starkexApiKey: string
   readonly starkexApiDelayHours: number
```

Comment:
Do we need this now? We still need to define a delay for refreshing view, but this should be renamed now to `dailyViewRefreshDelayHours`